ably-iosTests/ARTTestUtil.m

```diff
@@ -229,16 +230,40 @@ + (void)publishEnterMessages:(NSString *)clientIdPrefix count:(int) count channe
 
 + (void)testRest:(ARTRestConstructorCb)cb {
     [ARTTestUtil setupApp:[ARTTestUtil clientOptions] cb:^(ARTClientOptions *options) {
-        ARTRest * r = [[ARTRest alloc] initWithOptions:options];
-        cb(r);
+        ARTRest *rest = [[ARTRest alloc] initWithOptions:options];
+        cb(rest);
     }];
 }
 
-+(void) testRealtime:(ARTRealtimeConstructorCb)cb {
++ (void)testRealtime:(ARTRealtimeConstructorCb)cb {
     [ARTTestUtil setupApp:[ARTTestUtil clientOptions] cb:^(ARTClientOptions *options) {
-        ARTRealtime * realtime = [[ARTRealtime alloc] initWithOptions:options];
+        ARTRealtime *realtime = [[ARTRealtime alloc] initWithOptions:options];
         cb(realtime);
     }];
 }
 
++ (void)testRealtimeV2:(XCTestCase *)testCase withDebug:(BOOL)debug callback:(ARTRealtimeTestCallback)callback {
```

Comment:
We could really do with a comment explaining why we have `testRealtime` and `testRealtimeV2`